client/src/ajax.js

```diff
@@ -4,7 +4,7 @@ const BASE_URL = process.env.BASE_URL ? process.env.BASE_URL : '';
 
 const get = (id, cb) => {
   $.ajax({
-    url: `${BASE_URL}/restaurants/${id}`,
+    url: `/restaurants/${id}`,
     method: 'GET',
```

Comment:
why did you remove the base URL?

Reply:
the ajax url should be a relative url, so ${BASE_URL} is not required.